api/src/main/java/org/pmiops/workbench/dataset/DataSetMapper.java

```diff
@@ -3,14 +3,29 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.pmiops.workbench.db.model.DbDataDictionaryEntry;
+import org.pmiops.workbench.db.model.DbDataset;
+import org.pmiops.workbench.db.model.DbStorageEnums;
 import org.pmiops.workbench.model.DataDictionaryEntry;
+import org.pmiops.workbench.model.DataSet;
+import org.pmiops.workbench.model.PrePackagedConceptSetEnum;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 
 @Mapper(
     componentModel = "spring",
     uses = {CommonMappers.class})
 public interface DataSetMapper {
 
+  @Mapping(target = "id", source = "dataSetId")
+  @Mapping(target = "conceptSets", ignore = true) // This is stored as a list of ids. Look those up in the controller for entities
+  @Mapping(target = "cohorts", ignore = true) // This is stored as a list of ids. Look those up in the controller for entities
+  @Mapping(target = "domainValuePairs", ignore = true) // This is stored in a subtable, we may not want to fetch all the time
```

Comment:
OK. I think what we want then is another target class, such as DatasetLight. A mapper with a target class of `DataSet` should be expected to do whatever it takes to build a fully populated `DataSet`.